netlify/edge-functions/eleventy-edge.js

```diff
@@ -167,7 +156,7 @@ export default async (request, context) => {
                         flags = getExistingFeatureFlags(context)
                     }
                     // set cookies to pass data to client PostHog for bootstrapping
-                    if (flags[flag] && flags[flag] === value) {
+                    if (flags && flags[flag] && flags[flag] === value) {
```

Comment:
Why wouldn't line 151 be: `let flags = {}`?

Reply:
What codePath does actually require this change?

Reply:
`getExistingFeatureFlags(context)` returns `null` if there are no flags. That was tripping this when I ran it locally, so put this check in as a fix. I generally prefer returning `null` if there is no value, rather than an empty object.